app/views/content/non-uk-teachers/non-uk-qualifications.md

```diff
@@ -0,0 +1,135 @@
+
+This is a certificate costing £49.50 + VAT that proves your school and university qualifications are the same standard as UK GCSEs and a UK bachelor's degree. It takes 15 working days to complete your order from the date ENIC receives your qualification documents and payment.
+
+When you apply for teacher training, you'll be asked to submit details of your statement of comparability. Applications which include this are 28% more likely to be successful. 
+
+
+## Get help checking your qualifications
+
+You can also contact us for advice about your qualifications. Before applying, you can:
+
+* sign up to [get a teacher training adviser](/teacher-training-adviser/sign_up/identity) – you’ll be asked for your phone number, email and location so a dedicated adviser can call you back in your time zone
+* call +44 800 389 2500, Monday to Friday between 8:30am and 5:30pm (UK local time) except on [bank holidays](https://www.gov.uk/bank-holidays) in England - calls will be charged at your standard rate
+* use the live chat service at the bottom of this page
+
+We can help you with a free digital e-statement of comparability, once you’ve submitted your application, if your provider asks for this.
+
+
+## Qualifications you will need to train to teach
+
+To train to teach in England, you will need:
+
+* a degree which meets the standard of a UK bachelor’s degree (this can come from a non-UK university)
+* an English language qualification which meets the standard of a grade 4 General Certificate of Secondary Education (GCSE)
+* a maths qualification which meets the standard of a grade 4 GCSE
+* if teaching children aged 3 to 11 in primary school, a physics, chemistry or biology (science) qualification which meets the standard of a grade 4 GCSE
+
+A GCSE is an academic qualification awarded for exams in England, usually taken at age 16.
+
+You will also need to:
+
+* have the health and physical capacity to train to teach
+* undergo safeguarding and criminal record checks carried out by your training provider or employer
+
+## English language, maths and science proficiency
+
+Your English language qualification could be evidence you’ve reached a certain standard in an English language test, for example one of the following: 
+
+* the [International English Language Testing System (IELTS)](https://www.ielts.org/)
+* [Test of English as a Foreign Language (TOEFL)](https://www.ets.org/toefl)
+* [C2 Proficiency, formerly known as Cambridge English: Proficiency (CPE)](https://www.cambridgeenglish.org/exams-and-tests/proficiency/)  
+
+If you haven’t passed an English language test like this, or do not have qualifications in maths or science which are the same standard as grade 4 GCSE, some teacher training providers will let you sit tests in these subjects. It’s best to contact them to ask about their policy before you apply. 
+
+## If you do not have a university degree
```

Comment:
@laurahermionetennant92 we say bachelor's degree on our domestic pages rather than university degree